File: src/extension/compass/__tests__/fixtures/sample_gaps.ts
export class GapWidget {
	render(): number {
		return 1;
	}
}

declare const ns: { Service: new () => object };
declare const Registry: { lookup(n: number): number };

export function buildAll(): GapWidget {
	const widget = new GapWidget();
	const svc = new ns.Service();
	void svc;
	widget.render();
	Registry.lookup(1);
	return widget;
}

File: src/extension/compass/__tests__/fixtures/sample_gaps.kt
package demo

class GapWidget(val id: Int) {
    fun render(): Int {
        val service = GapService()
        service.transform(id)
        Registry.lookup(id)
        helperFun()
        return id
    }
}

class GapService {
    fun transform(value: Int): Int = value
}

object Registry {
    fun lookup(id: Int): Int = id
}

fun helperFun() {}
